app/templates/components/collapsed-objectives-program-years.hbs

```diff
@@ -0,0 +1,69 @@
+<div {{action @expand}} class="title" role="button">
+  {{t "general.objectives"}} ({{get (await this.objectives) "length"}})
+</div>
+
+{{#if (and
+  (is-fulfilled this.objectives)
+  (is-fulfilled this.objectivesWithMesh)
+  (is-fulfilled this.objectivesWithCompentency))}}
+  <div class="content">
+    <table>
+      <thead>
+        <tr>
+          <th class="text-left">{{t "general.summary"}}</th>
+          <th class="text-center">{{t "general.linkedCompetencies"}}</th>
+          <th class="text-center">{{t "general.meshTerms"}}</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            {{t "general.objectiveCount"
+              count=(get (await this.objectives) "length")}}
+          </td>
+          <td class="text-middle text-center" rowspan="3">
+            {{#if (eq
+              (get (await this.objectivesWithCompentency) "length")
+              (get (await this.objectives) "length"))}}
+              {{fa-icon "circle" class="yes"}}
+            {{else if (gte (get (await this.objectivesWithCompentency) "length") 1)}}
+              {{fa-icon "circle" class="maybe"}}
+            {{else}}
+              {{fa-icon "ban" class="no"}}
+            {{/if}}
+          </td>
+          <td class="text-middle text-center" rowspan="3">
+            {{#if (eq
+              (get (await this.objectivesWithMesh) "length")
+              (get (await this.objectives) "length"))}}
+              {{fa-icon "circle" class="yes"}}
+            {{else if (gte (get (await this.objectivesWithMesh) "length") 1)}}
+              {{fa-icon "circle" class="maybe"}}
+            {{else}}
+              {{fa-icon "ban" class="no"}}
+            {{/if}}
+          </td>
+        </tr>
+        <tr>
+          <td>
+            &nbsp;
```

Comment:
i don't get those non-breaking spaces. are you indenting here? if so - no need, get rid of them.

Reply:
I didn't add this part. This was there before I started working on it.